### DISCUSSION TITLE
Close : Release Notes, Documentation, and Users

### DISCUSSION THREAD

**Discussion by meteos77** [textbrowser/biblioteq#187] [General]
to answer your question: "Did you read the release notes?

first of all I recognize that you present the details in the release notes of BiblioteQ; better than google :-)

observation
as a user I almost never read the documentation provided with the objects I buy, or the software I download (at least as long as I don't encounter a dead end)
it's true for a washing machine or a car but also for LibreOffice or BiblioteQ and I'm not the only one (survey with my relatives)

**Comment by meteos77** [comment]
suggestions : 
some software present at startup a window with tips or advices (with a check mark "deactivate at next startup")
why not use this way of doing "at each new version of BiblioteQ" to present the Release Notes to all
this will allow to make the release notes known and to highlight your work with each new version

the 2nd problem of the release notes is that they are not in the user's language, which does not encourage non-English speakers to read them.

**Reply by textbrowser** [reply]
Notifications create distractions, even if they are optional. There is the notification itself which requires a new bunch of changes. Then there is the new option to re-enable the notification if it's been disabled. Then, how do I enable notification? I prefer exposing BQ so that it remains accessible and clear. If you want to read the release notes, read the release notes. If you do not and are encountering problems because you didn't read them, that's resolved in the next release through a persistent reminder.

I'm not creating translations. Interested people should do that, spend their time, and I'll spend my time integrating them.

**Reply by textbrowser** [reply]
Everyone should read documentation and find the time to learn about the product.

**Reply by textbrowser** [reply]
Some notifications are important. For example, you've lost a database connection. Notifications are difficult, generally. The file system is full, should I notify the database owner? You're consuming too many resources, notification? The permissions on your database file have changed, notification? Etc. These are specific problems that the application can report asynchronously (as they happen) or synchronously (when the person does something and the application cannot perform the task because of something). I have several projects with deep notifications and these things are not simple. They have to be intelligent on the CPU, on memory, on portability, on correctness, on availability of the things they're monitoring, etc. Then, they may never be used because the particular system doesn't have the problems. So, all of the work dies silently.

**Reply by meteos77** [reply]
between highlighting the release notes and all the notifications you describe, there is a gap :-)

**Reply by textbrowser** [reply]
Yes, the silence. :)

**Comment by meteos77** [comment]
about the documentation on page 13 

Creating a PostgreSQL Database
BiblioteQ supports PostgreSQL 8.x, 9.x, and newer. Please note that PostgreSQL 8.x and 9.x are not
supported by PostgreSQL

I don't understand your sentence and not using PostgreSQL?
I am waiting for your precision to translate this part

**Comment by textbrowser** [comment]
PostgreSQL (the community of) has stopped working on 8.x and 9.x. BQ supports 8.x (loosely) and 9.x. This means that at some point I cannot support them if I add new functions which are not available in those versions. I do not know which functions would do that because I don't know what I'll add. :)

**Reply by meteos77** [reply]
ok I understood
you just ask BiblioteQ users to stay with the maintained PostgreSQL versions

**Reply by textbrowser** [reply]
Of course. Qt, PostgreSQL, SQLite, YAZ, etc.

**Reply by meteos77** [reply]
translator problem :-)
postgreSQL 8.x and 9.x : not supported or no longer supported

**Reply by textbrowser** [reply]
Or you can write: Please install the latest stable release of PostgreSQL.

**Reply by textbrowser** [reply]
Find versioning here: https://www.postgresql.org/support/versioning/.

**Reply by meteos77** [reply]
beautiful formulation because it applies all the time :-)

**Comment by textbrowser** [comment]
From postgresql_update_schema.sql:

/* PostgreSQL 9.5 or newer is required. */

ALTER TABLE item_borrower ENABLE ROW LEVEL SECURITY;
ALTER TABLE item_request ENABLE ROW LEVEL SECURITY;
ALTER TABLE member_history ENABLE ROW LEVEL SECURITY;
ALTER TABLE member_history_dnt ENABLE ROW LEVEL SECURITY;
CREATE POLICY item_borrower_biblioteq_patron_policy ON item_borrower TO biblioteq_patron USING (memberid = session_user);
CREATE POLICY item_borrower_policy ON item_borrower TO biblioteq_administrator, biblioteq_circulation USING (true);
CREATE POLICY item_request_biblioteq_patron_policy ON item_request TO biblioteq_patron USING (memberid = session_user);
CREATE POLICY item_request_policy ON item_request TO biblioteq_administrator, biblioteq_circulation, biblioteq_librarian USING (true);
CREATE POLICY member_history_biblioteq_patron_policy ON member_history TO biblioteq_patron USING (memberid = session_user);
CREATE POLICY member_history_policy ON member_history TO biblioteq_administrator, biblioteq_circulation, biblioteq_librarian, biblioteq_membership USING (true);
CREATE POLICY member_history_dnt_biblioteq_patron_policy ON member_history_dnt TO biblioteq_patron USING (memberid = session_user);
CREATE POLICY member_history_dnt_policy ON member_history_dnt TO biblioteq_administrator, biblioteq_circulation, biblioteq_membership USING (true);

**Comment by meteos77** [comment]
version 1.19 in French
thanks for the integration in the package
[BiblioteQ-Fr-1.19.odt.tar.gz](https://github.com/textbrowser/biblioteq/files/8331984/BiblioteQ-Fr-1.19.odt.tar.gz)
s

**Comment by meteos77** [comment]
the publication note in French since the beginning of the adventure :-)
[2022-03-23_biblioteQ-realease-notes-fr.txt.tar.gz](https://github.com/textbrowser/biblioteq/files/8331995/2022-03-23_biblioteQ-realease-notes-fr.txt.tar.gz)

**Comment by textbrowser** [comment]
Thank you very much.

After this release, I'll start a new super-batch-processor. It will be the most beautiful, most intuitive part of BQ. It'll feature scanning, downloading of data, patrons, etc. A super-subset thing. Probably will require months to complete.

**Comment by meteos77** [comment]
great to have an evolution concerning batch management

I'm looking at a publishing software to assist "scribus" to discover another field of computer science
BiblioteQ is my first exercise for inspiration :-)

[test-scribus-1.pdf.tar.gz](https://github.com/textbrowser/biblioteq/files/8333270/test-scribus-1.pdf.tar.gz)

**Comment by textbrowser** [comment]
publication is not capitalized : Notes de publication. Is this correct?

**Reply by meteos77** [reply]
French grammar
only the first word of a title or subtitle is capitalized in French (unlike English).

"Notes de publication" is is therefore correct

**Comment by textbrowser** [comment]
Added the French release notes. As the document lacks formatting, it looks quite awful in the document window.

![Screenshot_20220323_115444](https://user-images.githubusercontent.com/10701156/159741513-f4c22611-f7b5-4652-805a-9cd2faed59ba.png)

**Comment by textbrowser** [comment]
Like, please correct it. :)

**Reply by meteos77** [reply]
ok I'll take back the text in html

**Comment by textbrowser** [comment]
Why? Simply add HTML tags. I mean it won't be simple because there are many tags.

**Reply by meteos77** [reply]
sed ^ and $ helped me a lot :-)
but utf8 (I have problems with accents) so switch to ISO8859-15

[Release-Notes-French.html.tar.gz](https://github.com/textbrowser/biblioteq/files/8335000/Release-Notes-French.html.tar.gz)

**Comment by textbrowser** [comment]
Is nice now.
